src/backend/api/reviews.js

```diff
@@ -0,0 +1,117 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+const reviewsColumns = new Set([
```

Comment:
A small detail to consider: 
Since you use this for input validation, what if you want to change the input to the API but not change the table columns? 

I'd try to name this based on what it is used for, validation, not what it happens to be similar to right now.

Reply:
This naming part is always so hard😅